### DISCUSSION TITLE
No setting for Duplication Detection Job Concurrency

### DISCUSSION THREAD

**Discussion by lx05** [immich-app/immich#10242] [Q&A]
### The bug

I can't find a setting to change the job concurrency for the "duplicate detection" job. Right now, only one job runs parallel.

### The OS that Immich Server is running on

Ubuntu 22 LTS

### Version of Immich Server

V1.106.3

### Version of Immich Mobile App

V1.106.3

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
irrelevant
```

### Your .env content

```Shell
irrelevant
```

### Reproduction steps

```bash
1.go to Administration -> settings
2. go to Job concurrency
3. No setting for Duplicate Detection there
...
```

**Comment by mertalev** [comment]
This is intentional since the job isn't concurrency-safe.
